tests/integration/CMakeLists.txt

```diff
@@ -71,3 +77,16 @@ target_link_libraries(vk_icd PRIVATE Vulkan::Headers)
 # vk_layer.h
 add_library(vk_layer MODULE ../vk_layer.c)
 target_link_libraries(vk_layer PRIVATE Vulkan::Headers)
+
+# vulkan.hpp
+add_library(vk_hpp MODULE ../vk_hpp.cpp)
+target_compile_features(vk_hpp PUBLIC cxx_std_11)
+target_link_libraries(vk_hpp PRIVATE Vulkan::Headers)
+
+# vulkan.cppm
+if (VULKAN_HEADERS_ENABLE_MODULE)
+    add_library(vk_hpp_module MODULE ../vk_hpp_module.cpp)
+    target_compile_features(vk_hpp_module PUBLIC cxx_std_20)
```

Comment:
This line should not be needed. Linking against Vulkan::HppModule should automatically handle this.

`Vulkan::HppModule` sets `cxx_std_20` when they link against the library.

```suggestion
```